install_toolchain.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script installs and builds all the tools needed for building and running Concord-BFT
+# It is used for building a docker image that is used by CI/CD and may be used in the
+# development environment.
+# If you prefer working w/o docker at your dev station just run the script with sudo.
+# If you need to add any tool or dependency this is the right place to do it.
+# Please bear in mind that we want to keep our list of dependencies as small as possible.
+set -ex
+
+APT_GET_FLAGS="-y --no-install-recommends"
+WGET_FLAGS="--https-only"
+
+echo ca_certificate=/etc/ssl/certs/ca-certificates.crt > ~/.wgetrc
+
+# Install building tools
+install_toolchain() {
+    apt-get update && apt-get ${APT_GET_FLAGS} install \
+        autoconf \
+        automake \
+        bison \
+        build-essential \
+        clang-format \
+        clang-tidy \
+        curl \
+        flex \
+        gdb \
```

Comment:
Should gdb,gdbserver propagate to the production images?

Reply:
No, but it's a separate issue.

Reply:
previous concord-bft images (the one I'm currently using) didn't have them, so I thought that you introduced them here.

Reply:
I didn't add anything new